packages/x-charts/src/RadarChart/RadarGrid/CircularRadarGrid.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { useTheme } from '@mui/material/styles';
+import { RadarGridRenderProps } from './RadarGrid.types';
+
+export function CircularRadarGrid(props: RadarGridRenderProps) {
+  const { center, corners, divisions, radius } = props;
```

Comment:
Is `divisions` verified not to be below 1? Not sure if it would create issues when dividing by 0 or a negative number.